src/services/orderApi.js

```diff
@@ -1,7 +1,17 @@
 import Axios from "axios"
 
+export const subdomainUrl = async (email) => {
+  const api = `https://mypatronpay.us/api/tenant_domain/`
+  const response = await Axios.post(api, {
+    email: email
+  })
+  return response;
+}
+
+
 export const login = async (email, password) => {
-  const api = `https://tenant3.mypatronpay.us/api/token/`
+let subDom = localStorage.getItem("subDomain")
```

Comment:
instead of keeping subDomain in local-storage you can keep it in .env file or constants file.

Reply:
subdomain is not a constant value it changes with login for example if i login with my id it will be different and if some one else logs in with their id it will be different so thats why i was keeping it in local-storage